Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -35,7 +35,7 @@ def binary_setcap(self):
 
 class NPMAgentUtil(IOMSAgent):
     def binary_setcap(self, binaryPath):
-        if os.path.exists('/opt/microsoft/omsconfig/Scripts/NPMAgentBinaryCap.sh') and os.system('sudo /opt/microsoft/omsconfig/Scripts/NPMAgentBinaryCap.sh ' + binaryPath) == 0:
+        if os.path.exists(AGENT_SCRIPT_PATH) and os.system('sudo %s %s' %(AGENT_SCRIPT_PATH, binaryPath)) == 0:
```

Comment:
please add comments on the behavior of when the customer didn't update the sudoers file.